### PR TITLE
Annwyn - First Pass at Populating Evil Wizard Tower

### PR DESCRIPTION
**Pull request by Chilly021**
* Make all walls indestructible
* Add Segment
* Add Entities (including evil wizard)
* Add Location Spawns 
* Add Sub Regions for Floors 3 / 4 

image
image
image

### CODE REVIEW

File: Segments/Annwn.mapproj [hunk 1]
```diff
@@ -1,8 +1,257 @@
...
+    
+    public class Plant : CreatureEntity
+        {
+            public Plant()
+            {
+                Body = 224;
+                Alignment = Alignment.Chaotic;
+            }
+
+            public FrostGiantUndead(Serial serial) : base(serial)
```
Comment (sproatba) [inline Segments/Annwn.mapproj:210]: Plant instead of FrostGiantUndead

File: Segments/Annwn.mapproj [hunk 2]
```diff
@@ -62958,11 +63522,642 @@
...
+    {
+        new CreatureBasicAttack(16)
+    };
+
+    archer.Wield(new Crossbow());
+    archer.Equip(new PlatemailArmor());
+
+    archer.AddGold(146);
+  
+    return glamuzu;
```
Comment (sproatba) [inline Segments/Annwn.mapproj:63687]: return archer?

**Review by sproatba — CHANGES_REQUESTED** [review]
the middle section of the tower might need the left side shaved by 1 hex, you'll end up with a bunch of mobs stuck in 2 hexes sometimes that way.

File: Segments/Annwn.mapproj (hunk 2 above)
Comment (sproatba) [reply Segments/Annwn.mapproj:63687]: lol typo on the typo correction return archer;

File: Segments/Annwn.mapproj (hunk 1 above)
Comment (Chilly021) [reply Segments/Annwn.mapproj:210]: Good call.

Comment (Chilly021) [reply Segments/Annwn.mapproj:210]: Fixed.

File: Segments/Annwn.mapproj (hunk 2 above)
Comment (Chilly021) [reply Segments/Annwn.mapproj:63687]: Fixed

**Comment by Chilly021** [comment]
> the middle section of the tower might need the left side shaved by 1 hex, you'll end up with a bunch of mobs stuck in 2 hexes sometimes that way.

I see yea was thinking best way to handle it since it opens up to the outside.

**Comment by Chilly021** [comment]
Stepping out but corrections done @sproatba

**Review by sproatba — APPROVED** [review]

**Review by SewerCzar — APPROVED** [review]
Nice!

**Review by zrikzlok — APPROVED** [review]

**Pull request merged**